pos_bypass_global_discount/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Camptocamp (https://www.camptocamp.com).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "POS Bypass Global Discount",
+    "summary": "",
+    "version": "16.0.1.0.0",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/pos",
+    "license": "AGPL-3",
+    "category": "Point of Sale",
+    "depends": ["pos_loyalty"],
```

Comment:
why not depend on `base_global_discount`?

Reply:
(to benefit from existing fields on product)

Reply:
Hi anh, it would be confusing imo. If we depends on base_global_discount, people may think those discounts are the same, in fact they are not (creating discount in base_global_discount does not mean it is used in POS). Also, others may not need base_global_discount 

To benefit from existing field on product, we can create a script to update the value, do you think so?

Reply:
what about this:
- add depends on `sale_global_discount`
- make it very explicit in the readme that the goal of this module is just to make sure POS will also honor `bypass_global_discount = True` when configured on products
- with the nature of the Global Discount being the native Discount feature at Order level